keras/src/trainers/data_adapters/py_dataset_adapter.py

```diff
@@ -40,6 +40,12 @@ class PyDataset:
             multiprocessed setting.
             Reduce this value to reduce the CPU memory consumption of
             your dataset. Defaults to 10.
+        shuffle: Whether to shuffle the sample ordering at the end of
+            each epoch. This argument is passed to `model.fit()`. When
+            calling `model.fit(..., shuffle=True)`, the training loop
+            automatically calls `on_epoch_end()` at each epoch
+            boundary, allowing datasets to implement custom
+            shuffling logic. Defaults to `False`.
```

Comment:
The `shuffle` argument is not part of `PyDataset.__init__`, so it should be removed from here. It's just an argument in your example.